modules/custom/dkan_metastore/src/Service.php

```diff
@@ -286,8 +293,7 @@ public function delete($schema_id, $identifier) {
    */
   private function objectExists($schemaId, $identifier) {
     try {
-      $this->getEngine($schemaId)->get($identifier);
-      return TRUE;
+      return $this->getEngine($schemaId)->get($identifier);
```

Comment:
I am not sure I like changing the semantics of this method to do more than just answer the question "does the object exist?".